model/database/cosmosdb-metrics.yaml

```diff
@@ -7,6 +7,12 @@ groups:
     unit: "{request_unit}"
     stability: experimental
     extends: attributes.db.cosmosdb.minimal
+    attributes:
+      - ref: db.cosmosdb.regions_contacted
+        brief: >
+          List of regions contacted by the client during a given operation.
```

Comment:
is overriding the brief needed here, or is it ok to inherit the brief in the registry?

Reply:
@sourabh1007 did you intend to resolve this discussion?

Reply:
yes i have made the changes.